### DISCUSSION TITLE
Not able to write in the Telegram channels

### DISCUSSION THREAD

**Discussion by GD-Dal** [xyzroe/XZG#55] [Q&A] [answered]
The admins of this group has restricted your ability to send messages

What must I do?

**Comment by xyzroe** [comment, marked as answer]
When you join a group you have 5 minutes to provide that you are human by clicking the button in a message from Rose. 
If you don't do this, you will be muted. 
You can try to rejoin the group.

**Reply by xyzroe** [reply]
@GD-Dal Have you fixed that problem?

**Reply by GD-Dal** [reply]
Yes.
But I wasn't able to complete all Rose wanted when using the web version.
As soon as I downloaded and used the desktop version (Windows 10), there was no problem.

So the web version of Telegram was the problem here
